app/js/app-menu.js

```diff
@@ -126,12 +159,19 @@ module.exports = function () {
     // time out before loading the default sample.
     // TODO search for a better way.
     setTimeout(function(){
-      $("#new-file").trigger('click');  
+      //check URL query
+      var url_path = getParameterByName('url');
```

Comment:
@ahmetcandiroglu please make sure that the query format is case insensitive (i.e. `uri`, `URI`, `url`, `URL` are valid queries)